CMakelists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.2)
```

Comment:
CMake 3.2 predates even C++17, so this minimum is bogus.

Reply:
I am sorry I had no idea that was the case. Thanks For Letting me know.